src/app/shell/provider/provider-config/provider-config.component.html

```diff
@@ -167,9 +164,9 @@
               </div>
             </ng-container>
             <div class="buttons2">
```

Comment:
we dont need 3 sets of buttons, move them outside individual step and use for whole stepper